bitnami/rabbitmq/templates/secrets.yaml

```diff
@@ -6,8 +6,8 @@ SPDX-License-Identifier: APACHE-2.0
 {{- $host := printf "%s.%s.svc.%s" (include "common.names.fullname" .) (include "common.names.namespace" .) .Values.clusterDomain }}
 {{- $port := print .Values.service.ports.amqp }}
 {{- $user := print .Values.auth.username }}
-{{- $password := include "rabbitmq.password" . }}
-{{- $erlangCookie := include "rabbitmq.erlangCookie" . }}
+{{- $password := include "common.secrets.passwords.manage" (dict "secret" (include "rabbitmq.secretPasswordName" .) "key" (include "rabbitmq.secretPasswordKey" .) "length" 16 "providedValues" (list "auth.password") "skipB64enc" true "skipQuote" true "context" $) }}
```

Comment:
Shouldn't you remove the `"skipQuote" true` parameter and directly use the resulting value at line 24 as you did for erlang cookie?

Reply:
Thanks for your review, @juan131.
I thought about it, but I'm not sure whether it would match its use further:
https://github.com/bitnami/charts/blob/5fb0e97d9336d40d86c3295637d4233218b8afea/bitnami/rabbitmq/templates/secrets.yaml#L63

The quotes might be problematic, aren't they?

Reply:
Ahh good catch! I haven't seen that block. Keep it as it's then. Thanks